testdata/dnn/download_models.py

```diff
@@ -889,6 +889,16 @@ def get_confirm_token(response):  # in case of large files
         url='https://drive.google.com/uc?export=dowload&id=19YWhArrNccaoSza0CfkXlA8im4-lAGsR',
         sha='1b4dd21a6baa5e3523156776970895bd3db6960a',
         filename='onnx/models/DB_TD500_resnet50.onnx'),
+    Model(
+        name='YuNet',
+        url='https://github.com/ShiqiYu/libfacedetection.train/raw/master/tasks/task1/onnx/yunet.onnx',
+        sha='49c52f484b1895e8298dc59e37f262ba7841a601',
+        filename='onnx/models/yunet.onnx'),
```

Comment:
> /master/

Use exact commit hash instead of `master`.

> onnx/models/yunet.onnx

If you plan to update model in the future (which is happened recently) please add something like `-YYYYMM` suffix to stored file name (e.g, `onnx/models/yunet-202109.onnx`).
Target location for `download_models.py` files is assumed as a shared storage without any support for versioning or file's updates. It is shared between different branches or OpenCV releases so its content should be persistent (you can add, but can't remove/replace).

Reply:
Sorry for the late response. These have been done in the latest commit.